app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">ParkingSystemJava</string>
+    <string name="configure_parking_size">configure parking size</string>
+    <string name="welcome">Welcome!</string>
 </resources>
```

Comment:
add an empty line at the end of the file to avoid this error

Reply:
done